qiskit/test/decorators.py

```diff
@@ -90,7 +90,9 @@ def _get_credentials():
                                                        discover_credentials)
     except ImportError:
         raise unittest.SkipTest('qiskit-ibmq-provider could not be found, '
-                                'and is required for executing online tests.')
+                                'and is required for executing online tests.'
```

Comment:
```suggestion
                                'and is required for executing online tests. '
```
When splitting strings across multiple lines (here and a few other places), there should be a trailing space.